### PR TITLE
Migration guide

### PR DESCRIPTION
**Pull request by zamotany**
#Ref: #933

### CODE REVIEW

File: docs/MigrationGuide.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+As described in previous section all apps resides inside `src/apps` directory. Each app has it's own folder, each one must have at least root component (by default it's `Index.js`), routes (`routes.js`) and reducers object (`reducers/index.js`).
+
+As you can see, its similar to GlueStick 0.x project structure, so you will only need to update paths in `import` statements and reducers index.
+
+For example, if previously there was:
+```
+import SomeComponen from 'components/SomeComponent';
+```
+now you must specify *who onws this component* - whether it's `shared` or inside some app:
```
Comment (dratwas) [inline docs/MigrationGuide.md]: typo in "owns"

File: docs/MigrationGuide.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+```
+import SomeComponen from 'shared/components/SomeComponen';
```
Comment (dratwas) [inline docs/MigrationGuide.md]: typo: `Component` the same is below ")

File: docs/MigrationGuide.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+- `<appName>` - path to `appName` app
+- `root` - path to project root directory
+- `src` - path to project source directory
+- `assets` - path to assets directory (`<root>/assets`)
+- `shared` - path to shared code (`<src>/shared`)
+- `config` - path to configurations (`<src>/config`)
+- `apps` - path to apps (`<src>/apps`)
+- `compiled` - path to `node_modules` module which needs to be parsed by webpack loaders (by default all modules from `node_modules` will be marked as external in server bundle, which means they won't be parsed by loaders, so in case you need module to be transpiled or if it's css/scss file you need to prefix it with this alias eg: `import 'compiled/normalize.css'`)
+
+Previusly `reducers/index.js` file was re-exporting reducers:
```
Comment (dratwas) [inline docs/MigrationGuide.md]: `Previously`

**Review by dratwas — CHANGES_REQUESTED** [review]

**Comment by trungly** [comment]
I had a lot of grammar suggestions, but they were too numerous and minor.  Instead I just pushed a commit to fix those.  Hope that's okay.  I believe all 3 of the typos pointed out by @dratwas are included.

**Comment by saltycrane** [comment]
Did you use https://github.com/skovhus/jest-codemods to migrate tests to Jest? Are there any gotchas we should look out for?

**Comment by saltycrane** [comment]
Another change I didn't see: `promiseMiddleware` changed to use `payload` instead of `value`.

**Comment by zamotany** [comment]
@saltycrane Good catch, I forgot about it.

**Comment by saltycrane** [comment]
@zamotany Thank you for the migration guide. I was very helpful.

**Pull request merged**